jdaviz/configs/default/plugins/export/export.py

```diff
@@ -93,6 +93,7 @@ class Export(PluginTemplateMixin, ViewerSelectMixin, SubsetSelectMixin,
     # if selected subset is spectral or composite, display message and disable export
     subset_invalid_msg = Unicode().tag(sync=True)
     data_invalid_msg = Unicode().tag(sync=True)
+    format_invalid_msg = Unicode().tag(sync=True)

```

Comment:
since there are other formats as well, perhaps we should call this `subset_format_invalid_msg` (I know its long 😬 ) or possibly just merge the functionality with `subset_invalid_msg`?